documentation/tarron_notes.md

```diff
@@ -2,12 +2,33 @@
 
 ## Questions
 
---
+- [ ] In the following code, it looks like each badge will have a "user" property that will return a user. Since we decided to just have one badge and we'll tie that to all the users that have earned it, should we get rid of this?
+- [ ] Since our Functions code doesn't have access to create/delete users, will all that have to be done on the front end? For the firebase user only, not the firestore user.
```

Comment:
We might just have to figure out why the backend code can't add or delete users and resolve that issue. I might dig into that.